src/plugins/gopts/gopts_win32.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * @file
+ *
+ * @brief Source for opts plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#ifndef ELEKTRA_GOPTS_SYSCTL_H
+#define ELEKTRA_GOPTS_SYSCTL_H
+
+#include <string.h>
+#include <windows.h>
+
+#include <kdbhelper.h>
+
+static int loadArgs (char *** argvp)
+{
+	int argc;
+
+	LPWSTR * args = CommandLineToArgvW (GetCommandLineW (), &argc);
```

Comment:
This implementation is a bit weird and I'm not entirely sure it will work if any non ASCII characters are passed as command-line arguments, but it is the best I could do.

The only alternative is using `GetCommandLine` (will use ASCII mode when `UNICODE` is not defined), but because there is no non-Unicode `CommandLineToArgv`, we would have to parse the string ourselves. Because ASCII mode in Windows isn't necessarily ASCII, but rather any 8-bit character encoding, that could be even more difficult than it already is (quotes, escaping, etc.)

Reply:
Should be okay for now.